types/evaluatex/index.d.ts

```diff
@@ -0,0 +1,33 @@
+// Type definitions for evaluatex 2.2
+// Project: https://github.com/arthanzel/evaluatex
+// Definitions by: Fawaz Orabi <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/evaluatex/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me